scripts/calculateRevenue.ts

```diff
@@ -41,12 +38,10 @@ async function main(args: ReturnType<typeof parseArgs>) {
       `Calculating revenue for ${userAddress} (${i + 1}/${eligibleUsers.length})`,
     )
 
-    const referralTimestamp = new Date(
-      timestamp - REFERRAL_TIME_BUFFER_IN_SECONDS,
-    )
+    const referralTimestamp = new Date(timestamp)
```

Comment:
this doesn't look like i've fixed anything! but in the base branch i proposed that `timestamp` is now a ISO date string, so that it is easier to reason about when reviewing the data